GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/BasicFileSystemTests.cs

```diff
@@ -8,12 +8,17 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Threading;
 
 namespace GVFS.FunctionalTests.Tests.LongRunningEnlistment
 {
     [TestFixture]
     public class BasicFileSystemTests : TestsWithEnlistmentPerFixture
     {
+        private const int FileAttributeSparseFile = 0x00000200;
+        private const int FileAttributeReparsePoint = 0x00000400;
+        private const int FileAttributeRecallOnOpen = 0x00040000;

```

Comment:
As per Abhijeet's email, should this be `FILE_ATTRIBUTE_RECALL_ON_DATA_ACCESS` (`0x00400000`) ?

Reply:
Yep thanks!